.integration

```diff
@@ -1,4 +1,6 @@
 command = 
 settings_file_name = 
 default_schedule = 0 0/5 * * * ?
-read_from_stdout = false
\ No newline at end of file
+read_from_stdout = false
+1.0 = (,22.17.0]
```

Comment:
Is it possible to use placeholders here? Like `1.* = (,22.17.0]`

Reply:
No, we have to use a tag name here. if only ```1.*``` is a tag